mapbox/src/main/java/org/odk/collect/mapbox/MapboxMapFragment.kt

```diff
@@ -295,21 +295,23 @@ class MapboxMapFragment :
             MapUtils.createPointAnnotations(requireContext(), pointAnnotationManager, markers)
 
         val featureIds = mutableListOf<Int>()
-        pointAnnotations.forEach {
-            val featureId = nextFeatureId++
-            val markerFeature = MarkerFeature(
-                requireContext(),
-                pointAnnotationManager,
-                it,
-                featureId,
-                featureClickListener,
-                featureDragEndListener,
-                MapPoint(it.point.latitude(), it.point.longitude())
```

Comment:
Problem was here: we were recreating the point that we save on the `MarkerFeature` and leaving altitude and accuracy behind.